### DISCUSSION TITLE
Mail stuck in pending status.

### DISCUSSION THREAD

**Discussion by programmeddeath1** [postalserver/postal#2122] [Help with using Postal]
Hi I have setup the postal server as per the documentation and demo video.

I have setup the test route but on sending a test mail it gets stuck in queue in pending status.
![text_test](https://user-images.githubusercontent.com/44861370/185652334-aa3aa065-a5b1-4ae8-a870-9506503edeee.png)

I have setup the dns and test route .
![image](https://user-images.githubusercontent.com/44861370/185654458-e701c687-ac25-4905-90d1-3f08db370a89.png)

![screen2aug19](https://user-images.githubusercontent.com/44861370/185656991-458c83bb-6917-4c96-9bdd-c7fff9283f0e.png)
![Screen3aug19](https://user-images.githubusercontent.com/44861370/185657000-68588e3d-0831-4558-8b64-9c76058e62b5.png)

Both incoming and outgoing messages are stuck in pending status, where can i see the logs as to why it is getting stuck?

Can you help understand what more needs to be done? any help would be appreciated.

Thanks in advance!

**Comment by ajay01994** [comment]
@willpower232 , could you please help us.... looking forward

**Comment by willpower232** [comment]
You need to confirm via `postal status` that everything is running.
